src/handlers.js

```diff
@@ -7,9 +7,8 @@ exports.callSusiApi = (slots, session, response) => {
         viewCount;
 
     if(query === "help"){
-        var endpoint = "http://api.susi.ai/susi/chat.json?q="+"Start+chatting"; // ENDPOINT GOES HERE
-        
         http.get(endpoint, (response1) => {
```

Comment:
This will give an error you define the end point after it so there is no endpoint defined. 
Codacy will get fixed if you will define 
var endpoint; above and then just assign endpoint = something where it was defined before this commit.